source/guide.liquid

```diff
@@ -98,12 +98,7 @@ body
   font: 100% $font-stack
   color: $primary-color
 {%- endcapture -%}
-
-{%- assign parsedExample = captureCode | codeExample: true, 'scss' -%}
-{%- assign example_id = 1 -%}
-{%- assign ui_id = 1 -%}
-
-{% render 'code_example', code: parsedExample,example_id: example_id, ui_id: ui_id %}
+{% codeExample variablesCodeExample, 'variables' %}
```

Comment:
@jgerigmeyer here's my first pass at the `codeExample` short code approach. I used the example name to replace both the numerical example ID number and the ui ID number in links. 

I wanted to also avoid setting this codeExample variable above the short code call but kept getting errors because of the multi-line arg. The short code call worked fine if the variable was something like `{% codeExample "my-test-css", 'variables' %}`. Not sure if you have other workaround suggestions?

Reply:
@sanajaved7 This looks great! I think what you were missing was a [`pairedShortcode`](https://www.11ty.dev/docs/languages/liquid/#paired-shortcode), which allows for contents to be within opening and closing tags instead of the first argument. I adjusted that in https://github.com/oddbird/sass-site/pull/17/commits/59b9d0c9e70fd48961e1eaa9909347663a0e8009 and it seems to work 🎉